salsa_dancing_molecules/startup/startup_script.py

```diff
@@ -19,17 +21,41 @@ def start(conf_path):
     Args:
         args - argument object from argparse.
     """
-    config = read_configuration(conf_path)
+    config = read_configuration(args.config_path)
     for simulation_conf in config.values():
         work_path = simulation_conf["workspace_path"]
         custom_materials = simulation_conf["materials_path"]
```

Comment:
Now that we can choose to not use custom materials, we should check if this key is in simulation_conf, then run the corresponding code for creating custom materials if that is the case.

Reply:
Good catch! Fixed.